profiles/factories.py

```diff
@@ -1,3 +1,4 @@
+from unittest.util import strclass
```

Comment:
What is it? Is it used?

Reply:
Of course it's not used. That's because of autoimport.
Removed it.